### PR TITLE
google/knative-gcp mimics all repo level settings in knative

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

**Comment by adrcunha** [comment]
/hold

Please double-check that this is indeed desirable by the repo admins.

File: ci/prow/templates/prow_config_header.yaml [hunk 1]
```diff
@@ -100,6 +100,7 @@ tide:
     - do-not-merge/invalid-owners-file
   merge_method:
     knative: squash
+    google: squash
```
Comment (adrcunha) [inline ci/prow/templates/prow_config_header.yaml]: If `google/knative-gcp` can be used instead, it would be a better choice.

**Comment by chaodaiG** [comment]
/hold
wait for @n3wscott 's comment

**Comment by adrcunha** [comment]
/lgtm

@chaodaiG unhold if you're still OK with these changes.

File: ci/prow/templates/prow_config_header.yaml (hunk 1 above)
Comment (chaodaiG) [reply ci/prow/templates/prow_config_header.yaml]: done

**Pull request merged**